lib/rspec/hive/query_builder.rb

```diff
@@ -0,0 +1,72 @@
+require_relative 'query_builder/row_transformer'
+require_relative 'query_builder/null_strategy'
+require_relative 'query_builder/value_by_type_strategy'
+
+module RSpec
+  module Hive
+    class QueryBuilder
+      def initialize(schema, connection)
+        @schema = schema
+        @connection = connection
+        @partition = {}
+        @rows = []
+        @stubbing = false
+      end
+
+      def partition(partition)
```

Comment:
we seem to have both an attribute called `@partition` with a protected accessor and method called `partition`...